index.bs

```diff
@@ -1439,22 +1509,60 @@ object is discarded in the runtime, subsequent attempts to access it via the
 protocol will result in an error.
 
 <div algorithm>
-To get the <dfn>object id for an object</dfn> given a |session| and |object|:
+To get the <dfn>object id for an object</dfn> given a |realm| and |object|:
 
-1. If |session|'s [=object id map=] does not contain |object|, run the
+1. If |realm|'s [=object id map=] does not contain |object|, run the
    following steps:
 
-  1. Let |object id| be a new, unique, string identifier for |object|. If |object| is an
-     [=/element=] this must be the [=web element reference=] for |object|; if it's a {{WindowProxy}}
-     object, this must be the [=browsing context id=] for |object|.
+  1. Let |object id| be a new, unique, string identifier for |object|. If
```

Comment:
It seems more reasonable for the shared id to match the web element reference or window handle, rather than the object id.